repos/system_upgrade/common/tools/importrpmgpgkeys

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/bash -ef
+
+log_error() {
+  echo >&2 "Error: $1"
+}
+
+if [ "$#" -eq 0 ]; then
+    log_error "Missing the required path to the directory with trusted GPG keys."
+    exit 1
+elif [ "$#" -ge 2 ]; then
+    log_error "Expected only one argument, received $#. Possibly unescaped whitespaces? '$*'"
+    exit 1
+fi
+
+if [ ! -e "$1" ]; then
```

Comment:
Should this be `-d` to make sure it is a directory here?

Reply:
I was thinking about that. but the script works same for script & file as well, so I rather chose to check that it just exists. but, in case that "$1 is symlink, it will not work as expected, so I will update the find to cover symlinks. - it's not needed for us now, but when there is no more work, why not to keep it like that? but I am ok to switch it just to directory if wanted. just explaining my decision